### DISCUSSION TITLE
project size error

### DISCUSSION THREAD

**Discussion by ptarroso** [opengisch/QField#4183] [Q&A]
Hi

I have a single project in the free plan with 24 files. The project manager in QField Cloud says, '24 project files in total using 191.2 MB.' However, the total size of the 24 files is only 98.8 MB (sum of the file sizes as given by QFieldCloud). This is causing issues with the synchronization feature. How can I identify where the problem lies?

**Comment by BoswachterMarc** [comment]
Perhaps https://github.com/opengisch/QField/issues/4129

**Comment by ptarroso** [comment]
Thanks @BoswachterMarc ! It was indeed a backup qgs file that was very large!
